### ISSUE TITLE
Add ability to truncate long field values

### ISSUE THREAD

**Issue by loveyuxiao** [sixlive/nova-text-copy-field#11]
```
            TextCopy::make('link', function () {
                return url('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=' . $this->id);
            }
            )->exceptOnForms()
```
it display in index too long , can nova-text-copy-field display short text and copy the real text?

**Comment by sixlive** [comment]
@loveyuxiao I've started work on an implementation (#12). Take a look and see if this will fit your use case.

**Comment by sixlive** [comment]
Released as part of [v1.1.0](https://github.com/sixlive/nova-text-copy-field/releases/tag/v1.1.0)
